extended/src/main/java/apoc/vectordb/ChromaDb.java

```diff
@@ -0,0 +1,239 @@
+package apoc.vectordb;
+
+import apoc.Extended;
+import apoc.ml.RestAPIConfig;
+import apoc.result.ListResult;
+import apoc.result.MapResult;
+import apoc.util.UrlResolver;
+import org.apache.commons.collections4.CollectionUtils;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.security.URLAccessChecker;
+import org.neo4j.internal.kernel.api.procs.ProcedureCallContext;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Mode;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static apoc.ml.RestAPIConfig.*;
+import static apoc.util.ExtendedUtil.listOfMapToMapOfLists;
+import static apoc.util.MapUtil.map;
+import static apoc.vectordb.VectorDb.executeRequest;
+import static apoc.vectordb.VectorDb.getEmbeddingResultStream;
+import static apoc.vectordb.VectorDbUtil.*;
+import static apoc.vectordb.VectorEmbedding.Type.CHROMA;
+import static apoc.vectordb.VectorEmbeddingConfig.*;
+
+@Extended
+public class ChromaDb {
+
+    @Context
+    public ProcedureCallContext procedureCallContext;
+
+    @Context
+    public Transaction tx;
+
+    @Context
+    public GraphDatabaseService db;
+
+    @Context
+    public URLAccessChecker urlAccessChecker;
+
+    @Procedure("apoc.vectordb.chroma.createCollection")
+    @Description("apoc.vectordb.chroma.createCollection(hostOrKey, collection, similarity, size, $config)")
```

Comment:
can we have a bit better descriptions (for all the procedures), not just the signature again? otherwise the apoc.help output is not really informative if it shows the same content twice without a human description